### PR TITLE
Update Docker guide to include Amazon SES

### PR DESCRIPTION
**Pull request by rdmurphy**
When I originally punched in this quick guide, I for some reason left it to only detail how to use SendGrid, even though our usage of it leaned on Amazon SES.

I know very little of how ActionMailer (or Rails, or even Ruby, honestly) works, so maybe this is too specific. 😛 But I figured it may help someone else.

### CODE REVIEW

**Comment by tommeagher** [comment]
Awesome. Thanks, @rdmurphy!

File: DOCKER.md [hunk 1]
```diff
@@ -13,3 +13,17 @@ ADMIN_EMAILS=
 SENDGRID_USERNAME=
 SENDGRID_PASSWORD=
 ```
+
+If you would like to use [Amazon SES](https://aws.amazon.com/ses/) instead to send emails, you'll need a different set of environmental variables.
+
+```sh
+DATABASE_URL=
+SECRET_KEY_BASE=
+ADMIN_EMAILS=
```
Comment (GabeIsman) [inline DOCKER.md]: These top three are unrelated to the different email provider, no? I'm a bit confused by their inclusion here.

Comment (rdmurphy) [inline DOCKER.md]: True — I included them because the SendGrid example also had them, and I wanted it to super explicit which environmental variables are required in both cases.

It could instead be re-written to say:
1) You always need these
2) And then pick one of these sets

Although I personally do like that you can copy-paste the entire template of what you need in one go.

**Comment by tommeagher** [comment]
I pegged this note on the end to my comment late Friday, but @rdmurphy, we still have #32 open. Is there anything else more that you could include in the Docker guide for newcomers?

File: DOCKER.md (hunk 1 above)
Comment (GabeIsman) [inline DOCKER.md]: Ah I see, I wasn't looking at in context. Makes more sense. Thanks for coming back and adding this!
